src/website/shared/utils.py

```diff
@@ -16,16 +19,29 @@ def get_gh(per_page: int = 30) -> Github:
     gh_auth: Auth.Auth | None = None
 
     if credentials_dir is None:
-        logger.warn("No credentials directory available, using unauthenticated API.")
+        logger.warning("No credentials directory available, using unauthenticated API.")
     else:
-        logger.warn(f"Using credentials directory: {credentials_dir}")
+        logger.info(f"Using credentials directory: {credentials_dir}")
 
         try:
             with open(f"{credentials_dir}/GH_TOKEN", encoding="utf-8") as f:
                 gh_auth = Auth.Token(f.read().rstrip("\n"))
-                logger.warn("Using GitHub Token to connect to the API.")
+                logger.info("Using GitHub Token to connect to the API.")
         except FileNotFoundError:
-            logger.warn(
+            logger.debug(
```

Comment:
Why only debug and not warning or at least info?

Reply:
warning is spammy, it's completely valid not to have that file.
It's only when both options are exhausted this becomes a warning.